ToyBox/classes/MainUI/LevelUp.cs

```diff
@@ -84,6 +84,11 @@ public class LevelUp {
                     Space(25);
                     Label(("Experimental".cyan() + ": lets you select any feat ignoring prerequisites.".green()).localize());
```

Comment:
Unrelated, but I really don't like the current localization system.

Just removing the "Experimental" would create a new key in the loc file; removing old ones is only possible by manually removing them. Keeping track of loc files across multiple versions is also a pain.

It'd probably be better to have loc strings either be static members of a localization class or something.